From ec4609b70a7deebf6381297aae95dd4bf05bd008 Mon Sep 17 00:00:00 2001
From: Vladimir Serbinenko <phcoder@gmail.com>
Date: Sun, 20 Apr 2014 16:12:41 +0200
Subject: [PATCH 080/152] 	* grub-core/kern/misc.c (__bzero): Don't
 compile in GRUB_UTIL.

	Reported by: Yves Blusseau <blusseau@zetam.org>.
---
 ChangeLog             | 6 ++++++
 grub-core/kern/misc.c | 3 +++
 2 files changed, 9 insertions(+)

diff --git a/ChangeLog b/ChangeLog
index 73a8928..3b61c0a 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,9 @@
+2014-04-20  Vladimir Serbinenko  <phcoder@gmail.com>
+
+	* grub-core/kern/misc.c (__bzero): Don't compile in GRUB_UTIL.
+
+	Reported by: Yves Blusseau <blusseau@zetam.org>.
+
 2014-04-20  Piotr Krysiuk  <piotras@gmail.com>
 
 	* grub-core/lib/i386/relocator.c: Allow loading old kernels by placing
diff --git a/grub-core/kern/misc.c b/grub-core/kern/misc.c
index 54db2e1..c5c815d 100644
--- a/grub-core/kern/misc.c
+++ b/grub-core/kern/misc.c
@@ -542,6 +542,9 @@ memset (void *s, int c, grub_size_t n)
   return grub_memset (s, c, n);
 }
 
+#endif
+
+#if !defined(GRUB_UTIL) && defined(__APPLE__)
 void GRUB_BUILTIN_ATTR
 __bzero (void *s, grub_size_t n)
 {
-- 
1.9.3

